packages/valory/protocols/contract_api/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # ------------------------------------------------------------------------------
 #
-#   Copyright 2022 fetchai
+#   Copyright 2022 valory
```

Comment:
inconsistent with `Valory AG` used in other places, but I suppose author name cannot include a white space either

Reply:
could maybe fix this in update copyright headers, that is to say map "valory" to "Valory AG"

Reply:
I will address this on 343

Reply:
Ohk, so this is generated by the protocol generator so this is the right format.